src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/InitializeTestProjectAttribute.cs

```diff
@@ -36,12 +36,7 @@ public override void Before(MethodInfo methodUnderTest)
                 throw new InvalidOperationException($"This should be used on a class derived from {typeof(MSBuildIntegrationTestBase)}");
             }
 
-            MSBuildIntegrationTestBase.Project = ProjectDirectory.Create(_originalProjectName, _testProjectName, _baseDirectory, _additionalProjects, _language);
-#if NETCOREAPP
-            MSBuildIntegrationTestBase.TargetFramework = "net5.0";
```

Comment:
I updated the test infrastructure to match the improvements we made in blazor.build